x-pack/plugins/actions/server/sub_action_framework/sub_action_connector.ts

```diff
@@ -84,7 +84,7 @@ export abstract class SubActionConnector<Config, Secrets> {
   }
 
   private getHeaders(headers?: AxiosRequestHeaders): Record<string, AxiosHeaderValue> {
-    return { ...headers, 'Content-Type': 'application/json' };
+    return { 'Content-Type': 'application/json', ...headers };
```

Comment:
Was this intentional?

Reply:
Yes, before, the `Content-Type': 'application/json'  couldn't be overwritten.